js/modules/Forms/EditorController.js

```diff
@@ -1038,6 +1042,150 @@ export class GlpiFormEditorController
         }
     }
 
+    /**
+     * Check if a block is used in conditions and show modal if needed
+     *
+     * @param {string} type Type of block ('question', 'comment', 'section')
+     * @param {jQuery} block The block element to check
+     * @returns {boolean} True if the block can be deleted, false otherwise
+     */
+    #checkBlockConditionDependencies(type, block) {
```

Comment:
Small distinction here but aren't block only questions and comments ?
If this is also used on sections, which are block, the name should be more generic (`checkItemConditionDependencies` ?)